app/views/assessor_interface/professional_standing_requests/show.html.erb

```diff
@@ -4,7 +4,7 @@
 <h1 class="govuk-heading-xl">Verify LoPS</h1>
 
 <p class="govuk-body">
-  You need to request verification for this applications LoPS. Follow the steps below:
+  You need to request verification for this application's LoPS.
```

Comment:
```suggestion
  You need to request verification for this application’s LoPS.
```

Reply:
I can't see the difference for this change

Reply:
It's a different kind of apostrophe, it just looks nicer: https://cmosshoptalk.com/2019/03/19/smart-apostrophes-cmos-6-117/